data-serving/data-service/src/index.ts

```diff
@@ -26,6 +19,13 @@ import mongoose from 'mongoose';
 import swaggerUi from 'swagger-ui-express';
 import validateEnv from './util/validate-env';
 
+// Set up appmetrics-dash before importing additional dependencies.
```

Comment:
Sorry about this. FYI, if you save without formatting (ctrl + k -> s) it won't run the imports sort.

Unfortunately the order needs to be maintained for now, but I'll take care of this today.

Reply:
aha, that's the command I was looking for, thanks! Reverted.